Upstream-Status: Pending
diff --git a/board/raspberrypi/rpi/rpi.env b/board/raspberrypi/rpi/rpi.env
index 30228285ed..cb2ff7954d 100644
--- a/board/raspberrypi/rpi/rpi.env
+++ b/board/raspberrypi/rpi/rpi.env
@@ -5,9 +5,9 @@
 dhcpuboot=usb start; dhcp u-boot.uimg; bootm
 
 /* Environment */
-stdin=serial,usbkbd
-stdout=serial,vidconsole
-stderr=serial,vidconsole
+stdin=serial
+stdout=serial
+stderr=serial
 
 /* DFU over USB/UDC */
 #ifdef CONFIG_CMD_DFU
